CMakeLists.txt

```diff
@@ -201,6 +201,11 @@ elseif( CMAKE_SYSTEM_NAME MATCHES "Darwin" )
    endif()
    message( STATUS "  MacOS SDK: ${CMAKE_OSX_SYSROOT}" )
    message( STATUS )
+
+   if( USE_BREAKPAD )
+      set(CMAKE_XCODE_ATTRIBUTE_DEBUG_INFORMATION_FORMAT "dwarf-with-dsym")
+   endif()
```

Comment:
https://cmake.org/cmake/help/latest/prop_tgt/XCODE_ATTRIBUTE_an-attribute.html
That documentation page says to use this CMAKE variable as a last resort and with great caution.

Are we sure this is necessary to get usable debug symbols?

Reply:
https://docs.sentry.io/platforms/apple/dsym/#uploading-symbols-with-sentry-cli
this is the only option which works, known to me at least (with XCode, other generators will require another solutions)